warehouse/query-core/src/main/java/datawave/next/stats/StatUtil.java

```diff
@@ -14,9 +14,15 @@ public class StatUtil {
      */
     public static String formatNanos(long ns) {
         long millis = ns / 1_000_000;
```

Comment:
I recognize this is a draft PR, but I just wanted to suggest that rather than manually converting the numbers here, you could look into using the formatting methods available via `TimeUnit.NANOSECONDS`. For example: 
```java
long millis = TimeUnit.NANOSECONDS.toMills(ns);

long millis = TimeUnit
```
The TimeUnit class also provides methods for directly

Reply:
Updated in latest commit